flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/TaskExecutorPartitionInfo.java

```diff
@@ -17,8 +17,20 @@
 
 package org.apache.flink.runtime.io.network.partition;
 
+import org.apache.flink.runtime.jobgraph.IntermediateDataSetID;
+
 /**
  * Encapsulates meta-information the TaskExecutor requires to be kept for each partition.
  */
 public final class TaskExecutorPartitionInfo {
+
+	private final IntermediateDataSetID intermediateDataSetId;
+
+	public TaskExecutorPartitionInfo(IntermediateDataSetID intermediateDataSetId) {
```

Comment:
do we already envision more fields?
I was just wondering whether we could keep just `IntermediateDataSetID` and switch to `TaskExecutorPartitionInfo` later when needed

Reply:
yes, the job name, task name, creation date and similar.